partiql-test-data/fail/static-analysis/primitives/aggregate-function-call.ion

```diff
@@ -1,127 +1,127 @@
-semantic::{
+{
```

Comment:
Most of the code diff in this PR is due to changing the assertion symbol and removing the test annotation. Reviewers can mostly focus on the data model's ISL file, `PartiQLTestDataValidator.kt`, and the updated markdown files.